src/serve/functions.ts

```diff
@@ -31,10 +31,11 @@ module.exports = {
     // we can use the port argument. Otherwise it goes to hosting and
     // we use port + 1.
     if (options.port) {
-      if (options.targets && options.targets.indexOf("hosting") < 0) {
-        args.port = options.port;
-      } else {
+      const hostingRunning = options.targets && options.targets.indexOf("hosting") >= 0;
```

Comment:
Under what circumstances is `options.targets` falsey?

Reply:
hopefully none but `options` is an `any` containing many `any`s and I just sleep better at night like this since `options` is often mutated in random places.